disnake/threads.py

```diff
@@ -303,8 +310,13 @@ def category_id(self) -> Optional[int]:
 
     @property
     def created_at(self) -> datetime.datetime:
-        """:class:`datetime.datetime`: Returns the thread's creation time in UTC."""
-        return snowflake_time(self.id)
+        """
+        :class:`datetime.datetime`: Returns the thread's creation time in UTC.
+
+        .. versionchanged:: 2.4
+            If create_timestamp is provided by discord, that will be used instead of the time in the id.
```

Comment:
```suggestion
            If create_timestamp is provided by discord, that will be used instead of the time in the ID.
```
It's a really minor change, but it's consistent.